src/components/ReceiveInfo/ReceiveInfo.tsx

```diff
@@ -112,82 +32,113 @@ export const ReceiveInfo = () => {
     ? myReceived.get(currentEpoch.id) ?? []
     : (previousEpoch && myReceived.get(previousEpoch.id)) ?? [];
   const totalReceived = (gifts && iti(gifts).sum(({ tokens }) => tokens)) || 0;
+  const [mouseEnterPopover, setMouseEnterPopover] = useState(false);
+  const [mouseEnterTrigger, setMouseEnterTrigger] = useState(false);
+  const closePopover = () => {
+    setMouseEnterPopover(false);
+  };
 
   return (
-    <div className={classes.root}>
-      <Button
-        color="surface"
-        size="small"
-        onClick={event => setAnchorEl(event.currentTarget)}
-      >
-        {totalReceived} {selectedCircle?.tokenName}
-      </Button>
-      <Popover
-        anchorEl={anchorEl}
-        anchorOrigin={{
-          vertical: 'bottom',
-          horizontal: 'right',
-        }}
-        classes={{
-          paper: classes.popover,
-        }}
-        onClose={() => setAnchorEl(undefined)}
-        open={!!anchorEl}
-        transformOrigin={{
-          vertical: 'top',
-          horizontal: 'right',
-        }}
+    <Popover open={mouseEnterPopover || mouseEnterTrigger}>
+      <PopoverTrigger>
+        <Button
+          size="small"
+          color="surface"
+          css={{ mr: '$sm' }}
+          onMouseEnter={() => setMouseEnterTrigger(true)}
+          onMouseLeave={() =>
+            setTimeout(() => setMouseEnterTrigger(false), 200)
+          }
+        >
+          {totalReceived} {selectedCircle?.tokenName}
+        </Button>
+      </PopoverTrigger>
+      <PopoverContent
+        onMouseEnter={() => setMouseEnterPopover(true)}
+        onMouseLeave={() => setTimeout(() => setMouseEnterPopover(false), 200)}
+        align="end"
+        sideOffset={-37}
```

Comment:
this magic number is interesting, is there any other way to do it?

Reply:
Sadly, I don't know how to get around this magic number.  It really stinks.  It's for the purpose of overlaying the popover and the trigger

Reply:
k